src/backend/src/modules/pia-intake/pia-intake.service.ts

```diff
@@ -263,7 +264,9 @@ export class PiaIntakeService {
         whereClause.push({
           ...commonWhereClause,
           ministry: In(mpoMinistries),
-          status: Not(PiaIntakeStatusEnum.INCOMPLETE),
+          status:
+            Not(PiaIntakeStatusEnum.INCOMPLETE) ||
+            Not(PiaIntakeStatusEnum.COMPLETE),
```

Comment:
This will simply resolve to `Not(PiaIntakeStatusEnum.INCOMPLETE)` with Javascript OR in place.